src/main/kotlin/me/rafaelldi/aspire/workload/AspireWorkloadService.kt

```diff
@@ -66,7 +66,7 @@ class AspireWorkloadService(private val project: Project, private val scope: Cor
                         .addAction(object :
                             NotificationAction(AspireBundle.message("notifications.go.to.documentation")) {
                             override fun actionPerformed(e: AnActionEvent, notification: Notification) {
-                                BrowserUtil.open("https://learn.microsoft.com/en-us/dotnet/aspire/whats-new/preview-4")
+                                BrowserUtil.open("https://learn.microsoft.com/ru-ru/dotnet/aspire/whats-new/preview-5")
```

Comment:
Is switching to Russian intended?

Sorry for the random "review", I was checking out what has been happening lately and came across this... So figured I might as well leave a comment :)

Reply:
No, good catch! Thank you!